_posts/2021-01-18-changelog-139.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #139"
+date: 2021-01-18 17:00:00 +0300
+---
+
+
+## New Features
+
+* Cargo features and `cfg` support improvements:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6497 -->
+    * Take into account attributes under [`cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) attribute during type inference, name resolution, and other analysis.
+      Now code insight works much better with code containing attributes like `#[cfg_attr(..., derive(...))]`, `#[cfg_attr(..., path="...")]`, etc.
```

Comment:
Now the corresponding features works as expected with code containing attributes like `cfg_attr(..., derive(...))`, `cfg_attr(..., path="...")`, etc